### DISCUSSION TITLE
Option for selecting between yml and properties formate

### DISCUSSION THREAD

**Discussion by KushParsaniya** [spring-io/start.spring.io#1687] [Ideas]
It will be good to have option for selecting the format of the application.properties file like Yaml or Properties . This feature would be similar to the existing option for selecting the packaging format of the project (e.g., JAR or WAR).

**Comment by mhalbritter** [comment]
Hey - this could be a candidate for https://github.com/spring-io/start.spring.io/issues/1679.

**Reply by KushParsaniya** [reply]
Yes.
